G3Common/src/users/BusinessCustomer.java

```diff
@@ -104,4 +101,20 @@ public void setBudgetMaxAmount(int budgetMaxAmount) {
 		this.budgetMaxAmount = budgetMaxAmount;
 	}
 
+	public int getBusinessW4cCode() {
+		return businessW4cCodeNumber;
+	}
+
+	public void setBusinessW4cCode(int businessW4cCodeNumber) {
+		this.businessW4cCodeNumber = businessW4cCodeNumber;
+	}
+	
+	/**
+	 * This is the toString for this class
+	 */
+	@Override
+	public String toString() {
+		return "Business Customer: "  + userFirstName + " " + userLastName + ", user ID = " + userId;
```

Comment:
Same as before, remove user id and put company name instead

Reply:
fixed